shared_model/interfaces/query_responses/transactions_page_response.hpp

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef IROHA_SHARED_MODEL_TRANSACTIONS_PAGE_RESPONSE_HPP
+#define IROHA_SHARED_MODEL_TRANSACTIONS_PAGE_RESPONSE_HPP
+
+#include "interfaces/base/model_primitive.hpp"
+#include "interfaces/common_objects/range_types.hpp"
+#include "interfaces/common_objects/types.hpp"
+
+namespace shared_model {
+  namespace interface {
+    /**
+     * Container of asset, for fetching data.
+     */
+    class TransactionsPageResponse
+        : public ModelPrimitive<TransactionsPageResponse> {
+     public:
+      /**
+       * @return Attached transactions
+       */
+      virtual types::TransactionsCollectionType transactions() const = 0;
+
+      virtual interface::types::HashType nextTxHash() const = 0;
```

Comment:
Would be nice to add a method to check whether there are more transactions following, like
```
bool isLastPage() const;
```
which could be defined in `shared_model/interfaces/query_responses/impl/transactions_page_response.cpp`.

Reply:
Where such method can be used? I think now it is enough to check by next tx hash method, which returns none when the page is the last